crates/miden-lib/asm/miden/note.masm

```diff
@@ -225,3 +226,32 @@ export.extract_sender_from_metadata
     swap
     # => [sender_id_prefix, sender_id_suffix]
 end
+
+#! Computes the tag for a network note for a given network account such that it is picked up by the network transaction builder.
+#!
+#! This procedure implements the same logic as in Rust in NoteTag::from_network_account_id().
+#!
+#! Inputs:  [account_id_prefix, account_id_suffix]
+#! Outputs: [network_account_tag]
+#!
+#! Where:
+#! - account_id_prefix, account_id_suffix is the account id to compute the note tag for.
+#! - network_account_tag is the computed network note tag.
+#!
+#! Invocation: exec
+export.get_network_account_tag
```

Comment:
nit: I would rename this to `build_note_tag_for_network_account` or something similar.

Also, should we check if the account is actually a network account? Or at least I would update the comments to indicate that we don't check for this.